astrokat/simulate.py

```diff
@@ -135,6 +136,7 @@ def __iter__(self):
         raise StopIteration
 
     def __exit__(self, type, value, traceback):
+        # TODO: what is self.track_ used for again?
```

Comment:
This is a bit concerning - would prefer all traces are removed, if there is no purpose.

Reply:
This is for multiple observation loops, needed for zoom modes
I never got time to finish the implementation, sort of like the correlator setup implementation

These are the 2 big hanging implementations that I still need to do and will add separate branches to fix them